src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,36 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
+    private final int MINIMAL_AGE = 18;
+    private final int MINIMAL_LENGTH = 6;
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws ImpossibleRegisterUserException {
+        if (user == null) {
+            throw new ImpossibleRegisterUserException("The user can`t be empty");
```

Comment:
The exception message 'The user can`t be empty' could be more informative. Consider specifying that a null user object was provided.